ldap/servers/slapd/log.c

```diff
@@ -193,6 +193,19 @@ compress_log_file(char *log_name)
         gzclose(outfile);
         return -1;
     }
+
+    /* Close the gzip file and set the perms, then start writing to it */
+    gzclose(outfile);
+    if (chmod(gzip_log, mode) != 0) {
```

Comment:
I have a concern about that one: 
  The file is still created world writable and someone may open the file before the chmod is done.
  I am afraid that the only safe way is to use umask so that gzopen creates the file with the proper mode
  but as said in the man page, umask is prone to race condition.   Maybe we should have global a 0177 umask and explicitly set the proper mode after creating the files  everywhere else (A least if we forget to change some mode it will not be any more a security issue)

Reply:
It's not world writable, only readable:

    -rw-r--r--. 1 dirsrv dirsrv    900 Jul 15 17:00 access.20250715-165309.gz

So I would think this approach is still valid.

Reply:
@progier389 - ok changes made please review...